From df267e6b1b4c2116001c6c64c71f01c14d78b572 Mon Sep 17 00:00:00 2001
From: Olivier BIDEAU <olivier.bideau@st.com>
Date: Tue, 14 Nov 2023 14:27:25 +0100
Subject: [PATCH 428/954] feat(fdts): activate IWDG1 on STM32MP257F-DK

Enable IWDG1 watchdog on STM32MP257F-DK board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git edf90c6c5e64f76d6bc03fa95bdd84cdbb24850c feat(fdts): activate IWDG1 on STM32MP257F-DK]
Signed-off-by: Olivier BIDEAU <olivier.bideau@st.com>
Change-Id: If638af7f8fe8d38a6dcd960c2bc4189035ac6704
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/340211
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp257f-dk.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/fdts/stm32mp257f-dk.dts b/fdts/stm32mp257f-dk.dts
index 92e7c439ec..d6e140c3c6 100644
--- a/fdts/stm32mp257f-dk.dts
+++ b/fdts/stm32mp257f-dk.dts
@@ -155,6 +155,11 @@
 	};
 };
 
+&iwdg1 {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &pwr {
 	vddio1: vddio1 {
 		vddio1-supply = <&vddio_sdcard>;
-- 
2.53.0

